src/components/Contribute/Knowledge/UploadFile.tsx

```diff
@@ -126,6 +131,14 @@ export const UploadFile: React.FunctionComponent<{ onFilesChange: (files: File[]
           titleTextSeparator="or"
           infoText="Accepted file types: Markdown"
         />
+        <p className="spinner-container">
+          {isUploading && (
+            <>
+              <Spinner size="lg" />
+              <p>Uploading file</p>
```

Comment:
@nikhil-krajput can you please update this text to "Uploading files to `taxonomy-knowledge-docs` repo in your github account."

Reply:
@vishnoianil 
Updated text:
![Screenshot 2024-10-18 at 14-14-06 InstructLab UI](https://github.com/user-attachments/assets/2153f6c4-6fdc-486c-9cb8-52bf72d95fa8)